app/assets/main/react_components/footprint_result/MembershipSelector.jsx

```diff
@@ -15,11 +15,13 @@ const MembershipSelector = ({ signUpText, grantedReferralCode, setGrantedReferra
                 multipleOffsets={multipleOffsets} setMultipleOffsets={setMultipleOffsets}
             />
             <div data-inactive-class="hidden">
+            { !(selectedMembership === "free") &&
```

Comment:
Kanske bara är olika "skolor", men vad jag sett brukar man skriva enligt förslag nedan. Då behöver man bara kolla "på ett ställe" för att förstå vilket condition som evalueras. Tankar?
```suggestion
            { selectedMembership !== "free" &&
```